dev-requirements.txt

```diff
@@ -261,7 +261,8 @@ nbformat>=5.9.0; python_version >= '3.8'
 notebook>=4.3.1,<6.1.0; python_version == '2.7'
 notebook>=6.4.10; python_version == '3.6'
 notebook>=6.5.4; python_version == '3.7'
-notebook>=7.0.2; python_version >= '3.8'
+notebook>=7.0.7; python_version >= '3.8'
```

Comment:
notebook 7.0.6 fixes the issue. I suggest to use the lowest version that fixes the issue.

Also in minimum-requirements.txt

Reply:
See comment above. The safety issue states affected spec: >= 7.0.0, <= 7.0.6 so the fixed version is 7.0.7.  They state "Patched versions include notebook above 7.0.7". 

However, there is no notebook 7.0.7. The next version after 7.0.6 is 7.1.0,  That is going to add more complexity in that it modifies minimums in other jupyter notebook components.